### ISSUE TITLE
WebId creation for TLS

### ISSUE THREAD

**Issue by nicola** [linkeddata/node-webid#7]
Implement generate using code from [openssToX509](https://github.com/gitpay/util/blob/master/opensshToX509.js)

**Comment by melvincarvalho** [comment]
Let me know if I can help with this! :)

**Comment by nicola** [comment]
Actually, please do!

the idea is to implement:

```
function generate(options, callback) {
    // where options has uri
}
```

**Comment by sible777** [comment]
Would you be opposed to using [pem](https://github.com/andris9/pem) to implement this feature? It shouldn't be too hard to just wrap it's functionality up to create the certificate. Can even add in the ability to generate a basic profile doc in RDF or just create the cert for an existing profile.

**Comment by sible777** [comment]
I've began implementing the generate function [here](https://github.com/sible777/node-webid/blob/provideWebid/tls/index.js#L104) using the pem module to create the certificate.

**Comment by nicola** [comment]
@sible777 this library should just create a certificate with `.generate`, since I will implement this under `tls/`

**Comment by nicola** [comment]
hey @sible777, can you ping me on gitter to update me on how I can take it forward?

**Comment by nicola** [comment]
also, I would prefer `node-forge` to `pem` :)

**Comment by sible777** [comment]
@nicola sure thing.

**Comment by dmitrizagidulin** [comment]
Looks like this issue can be closed? (with PR #9 having been merged)
